develop/tutorials/articles/02-from-liferay-6-to-liferay-7/07-upgrading-plugins-to-liferay-7/03-upgrading-the-liferay-maven-environment.markdown

```diff
@@ -0,0 +1,107 @@
+# Upgrading the Liferay Maven Environment
+
+If you're an avid Maven user, and have been using it for Liferay Portal 6.2
+project development, you must upgrade your Maven environment to be compatible
+with @product-ver@ development. There are two main parts of the Maven
+environment upgrade process that you must address:
+
+- Upgrading Liferay Maven Plugins
+- Upgrading Liferay Maven Artifacts
+
+The changes for both of these topics only occur in your parent POM and any
+relevant child project POMs. For more information on using Maven with
+@product-ver@, see the
+[Maven tutorial section](/develop/tutorials/-/knowledge_base/7-0/maven).
+
+You'll start off by upgrading your Maven environment's Liferay Maven plugins.
+
+## Upgrading Liferay Maven Plugins
+
+The biggest change for your project's build plugins is the removal of the
+`liferay-maven-plugin`. Liferay now provides several individual Maven plugins
+that accomplish specific tasks. For example, you can configure Maven plugins for
+Liferay's CSS Builder, Service Builder, Theme Builder, etc. With smaller plugins
+available to accomplish specific tasks in your project, you no longer have to
+rely on one bulky plugin that provides many things you may not want.
+
+As an example, suppose your Liferay Portal 6.2 project was using the
+`liferay-maven-plugin` to only use Liferay CSS Builder. First, you should remove
+the legacy `liferay-maven-plugin` plugin dependency from your project's
+`pom.xml` file:
+
+    <plugin>
+        <groupId>com.liferay.maven.plugins</groupId>
+        <artifactId>liferay-maven-plugin</artifactId>
+        <version>${liferay.version}</version>
+        <configuration>
```

Comment:
@gamerson Those appServer** config settings resided in the parent POM, right? Something like this? 

```
<properties>
       <liferay.app.server.deploy.dir>
         E:\liferay-portal-6.2.0-ce-ga1\tomcat-7.0.42\webapps
       </liferay.app.server.deploy.dir>

       <liferay.app.server.lib.global.dir>
         E:\liferay-portal-6.2.0-ce-ga1\tomcat-7.0.42\lib\ext
       </liferay.app.server.lib.global.dir>

       <liferay.app.server.portal.dir>
         E:\liferay-portal-6.2.0-ce-ga1\tomcat-7.0.42\webapps\root
       </liferay.app.server.portal.dir>

       <liferay.auto.deploy.dir>
         E:\liferay-portal-6.2.0-ce-ga1\deploy
       </liferay.auto.deploy.dir>

       <liferay.version>
         6.2.0
       </liferay.version>

       <liferay.maven.plugin.version>
         6.2.0
       </liferay.maven.plugin.version>
    </properties>
```